Documentation/nfs-crd.md

```diff
@@ -0,0 +1,196 @@
+---
+title: NFS Export
+weight: 38
+indent: true
+---
+
+# NFS Export CRD
+NFS Exports can be created and configured using the `export.nfs.rook.io` custom resource definition (CRD).
+Please refer to the [user guide walk-through](nfs.md) for complete instructions.
+This page will explain all the available configuration options on the NFS CRD.
+
+## Sample
+
+The parameters to configure NFS CRD are demonstrated in the example bellow which is followed by a table that explains the parameters:
+
+A simple example for sharing a volume(could be hostPath, cephFS, cephRBD, googlePD, EBS etc.) using NFS, without client specification and per export based configuration, whose NFS-Ganesha export entry looks like:
+```
+EXPORT {
+    Export_Id = 1;
+    Path = /export;
+    Pseudo = /nfs-share;
+    Protocols = 4;
+    Sectype = sys;
+    Access_Type = RW;
+    Squash = none;
+    FSAL {
+        Name = VFS;
+    }
+}
+```  
+the CRD instance will look like the following:
+```yaml
```

Comment:
Will it be fine if it is on both the pages?
On the first page, it can be `cat nfs.yaml` to demonstrate the content of the file
and on the other page, it can be used to describe the CRD instance

Reply:
yes, if the CRD is on both pages that will be helpful